client/client.go

```diff
@@ -264,15 +269,6 @@ func (c *client) tsLoop() {
 	}
 }
 
-func extractSpanReference(requests []*tsoRequest, opts []opentracing.StartSpanOption) []opentracing.StartSpanOption {
-	for _, req := range requests {
-		if span := opentracing.SpanFromContext(req.ctx); span != nil {
-			opts = append(opts, opentracing.ChildOf(span.Context()))
```

Comment:
Can there be scenario where for some requests span is nil?
In those scenario, current code may not work hence need to update it.